packages/core/src/codewhisperer/util/authUtil.ts

```diff
@@ -515,20 +491,6 @@ function buildFeatureAuthState(state: AuthState): FeatureAuthState {
     }
 }
 
-function getEnvType(): keyof HasAlreadySeenQWelcome {
```

Comment:
For reference: the replacement is `getComputeEnvType()` : https://github.com/aws/aws-toolkit-vscode/blob/7ec63ca3607e5d307f7f6fe83a2a1a51878e844c/packages/core/src/shared/telemetry/util.ts#L113